packages/core/src/repository/errors.ts

```diff
@@ -1,71 +1,83 @@
+
+export const UnknownErrorCode = -1;
+
+export const MethodNotImplementedErrorName =  'ethod Not Implemented';
```

Comment:
`M` is missing on the string. `ethod` => `Method`

Reply:
Excelent catch.

Reply:
Solved